src/AmalgamatorSession.ts

```diff
@@ -675,6 +696,37 @@ export class AmalgamatorSession extends LoggingDebugSession {
             ].customRequest('cdt-gdb-adapter/Memory', args);
             response.body = childResponse.body;
             this.sendResponse(response);
+        } else if (command === 'cdt-amalgamator/resumeAll') {
+            const [, threads] = await this.collectChildTheads();
+            for (const thread of threads) {
+                const [childIndex, childId] = await this.getThreadInfo(
+                    thread.id
+                );
+                const childDap = this.childDaps[childIndex];
+                const childResponse = await childDap.threadsRequest();
+                for (const threadOfChildDap of childResponse.body.threads) {
+                    const ThreadInfo = threadOfChildDap as ThreadInfo;
+                    if (ThreadInfo.running === false) {
+                        const args = {
+                            threadId: childId,
+                        } as DebugProtocol.ContinueArguments;
+                        const continueResponse = await childDap.continueRequest(
+                            args
+                        );
+                        if (continueResponse.body === undefined) {
+                            continueResponse.body = {};
+                        }
+                        continueResponse.body.allThreadsContinued = false;
+                        this.sendEvent(
+                            new ContinuedEvent(
+                                thread.id,
+                                continueResponse.body.allThreadsContinued
+                            )
+                        );
```

Comment:
Lets try to write this a bit more succinctly:

```suggestion
                        await childDap.continueRequest({ threadId: childId });
                        this.sendEvent(new ContinuedEvent(thread.id, false));
```

Reply:
I updated it. Thank you for your support.